x/group/client/cli/util_test.go

```diff
@@ -0,0 +1,32 @@
+package cli
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_ParseCLIProposal(t *testing.T) {
+	a := assert.New(t)
```

Comment:
Nit (not a blocker): I don't think there's much value to assigning assert to a variable.

Mainly, I'd prefer to just be consistent. Also, if do we assign to a variable, it should be `require`, not `assert`.